engine.py

```diff
@@ -222,7 +222,7 @@ def _remove_shotgun_menu(self):
         self.log_debug("Removing the shotgun menu from the main menu bar.")
         self._menu_generator.destroy_menu()
 
-    def _on_menus_loaded(self, code):
+    def _on_menus_loaded(self, code=None):
```

Comment:
Could we simply not pass in anything instead when calling it? It's an internal method after all.